seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -160,7 +160,7 @@
       "tests": [
         "assert(code.match(/const/g), 'message: Do not replace <code>const</code> keyword.');",
         "assert(code.match(/const\\s+s/g), 'message: <code>s</code> is declared with <code>const</code>.');",
-        "assert(code.match(/const\\s+s\\s*?=\\s*?\\[\\s*?2\\s*?,\\s*?5\\s*?,\\s*?7\\s*?\\]\\s*?;/g), 'message: Do not change the original array declaration.');",
+        "assert(code.match(/const\\s+s\\s*?=\\s*?\\[\\s*?5\\s*?,\\s*?7\\s*?,\\s*?2\\s*?\\]\\s*?;/g), 'message: Do not change the original array declaration.');",
```

Comment:
@Ethan-Arrowood I think this regex can safely be changed to:
```js
code.match(/const\\s+s\\s*=\\s*\\[\\s*5\\s*,\\s*7\\s*,\\s*2\\s*\\]\\s*;?/g)
```
The `?`s are redundant since `*` is already saying the space is optional, and also, this makes the semicolon optional since all we really care about is how the array was declared. 

Nice catch on the order of the array elements by the way! I stared at the regex locally for like 10 minutes before I realized it was the wrong order 😄

Reply:
@Ethan-Arrowood Yeah, I know you didn't write it 😄, just figured I'd point it out since you had a PR open.